lnbits/core/models.py

```diff
@@ -213,6 +223,18 @@ class Payment(FromRowModel):
     webhook: Optional[str]
     webhook_status: Optional[int]
 
+    @property
+    def success(self) -> bool:
+        return self.status == PaymentState.SUCCESS.value
+
+    @property
+    def failed(self) -> bool:
+        return self.status == PaymentState.FAILED.value
+
+    @property
+    def pending(self) -> bool:
+        return self.status == PaymentState.PENDING.value
+
```

Comment:
```suggestion

    def __str__(self):
        return str(self.value)
```

Reply:
https://stackoverflow.com/questions/58608361/string-based-enum-in-python#58608362
![screenshot-1716795178](https://github.com/lnbits/lnbits/assets/1743657/b47d5323-20f5-423b-8db5-9f7148c23a8c)

i think this is not needed ffor string based enums

Reply:
i changed it anyways :)